### PR TITLE
New presentation transition: DropDown

### PR DESCRIPTION
**Pull request by tbaranes**
Related to #198

### CODE REVIEW

File: IBAnimatable/DropDownAnimator.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    animatingView.layer.addAnimation(animation, forKey: "dropdown")
+  }
+
+  func animateDismissingDropDown(animatingView animatingView: UIView, completion: AnimatableCompletion) {
+    var point = animatingView.center
+    let angle = CGFloat(arc4random_uniform(100)) - 50
+    point.y += UIScreen.mainScreen().bounds.height
+    UIView.animateWithDuration(transitionDuration, animations: {
+      animatingView.center = point
+      animatingView.transform = CGAffineTransformMakeRotation(angle / 100)
```
Comment (JakeLin) [inline IBAnimatable/DropDownAnimator.swift:82]: `CGAffineTransformMakeRotation` is taking `radians` not `degrees`. Is it what you want here?

**Comment by JakeLin** [comment]
@tbaranes well done 👍, I play around it and it works fine, my only concern is `CGAffineTransformMakeRotation` (please the comment), I don't quite understand your algorithm. In my mind, it should have something like `randomNubmer * .pi`. But the effect is working fine. So the PR is all good.

**Comment by tbaranes** [comment]
I have to say I didn't look deeper since it was working as expecting 😬 
I'm gonna merge this now, if there any improvements necessary, we can still add them later!

**Comment by JakeLin** [comment]
All good @tbaranes thanks

**Pull request merged**